docusaurus.config.js

```diff
@@ -91,12 +91,13 @@ module.exports = {
               },
               {
                 tagName: 'script',
-                innerHTML: `
```

Comment:
mind bumping the sdk version to:

https://cdn.jsdelivr.net/npm/statsig-js@4.3.0/build/statsig-prod-web-sdk.min.js

while youre at it?